tests/directory.rs

```diff
@@ -79,6 +82,7 @@ fn directory_in_root() -> io::Result<()> {
 }
 
 #[test]
+#[ignore]
```

Comment:
These guys were failing for me locally but worked in the CI. Thought it was my local env or something, guess you had the same problem?

Reply:
You might be thinking of the git repo tests, which fail for me locally too:
https://github.com/starship/starship/blob/d36d232cc6a2987263dacbbe293af624d647cdb3/tests/directory.rs#L100-L134

I'm still not 100% sure why they fail locally. I have yet to use a debugger to step through those tests to see why they fail, but they seem to pass as expected in CI. 🤔 

These new `ignore` attributes were added because the directory is expected to exist for real, now that we read the directory on `Context` creation.
https://github.com/starship/starship/blob/d36d232cc6a2987263dacbbe293af624d647cdb3/src/context.rs#L26-L35

This wouldn't be a problem if we could mock the output of `fs::read_dir`, but I don't think Rust provides us with a very easy way to do that. 😅 

I'm going to create an issue where we can further discuss test strategy.